docs/zero-knowledge-glossary/index.mdx

```diff
@@ -1,6 +1,6 @@
 ---
-title: Zero knowledge glossary
-description: An open source glossary to aid your zero knowledge journey!
+title: Zero-knowledge glossary
+description: An open-source glossary to aid your zero-knowledge journey!
```

Comment:
```suggestion
description: An open-source glossary to aid your zero knowledge journey!
```

Again, I think the hyphen is unhelpful here. It suggests that the user is going on a journey armed with--literally--_zero knowledge_, rather than embarking on a journey learning about `zero knowledge` tech.